mscore/editstafftype.ui

```diff
@@ -651,6 +651,13 @@
                    </widget>
                   </item>
                   <item>
+                   <widget class="QCheckBox" name="showTabFingering">
+                    <property name="text">
+                     <string>Show fingerings</string>
```

Comment:
Better `Allow fingerings`. Applies to Tab staves only anyway and doesn't magically show or hide existing fingerings.